tests/test_loadDiaCatalogs.py

```diff
@@ -390,6 +395,25 @@ def testGetPixelRanges(self):
             self.assertEqual(normPix, testPix)
             self.assertEqual(flipPix, testPix)
 
+    def test_apdbSchema(self):
+        """Test that the DiaSource schema from dax_apdb agrees with the column
+        names defined here in ap_association/data/DiaSource.yaml.
+        """
+        functorFile = _data_file_name("DiaSource.yaml", "ap_association")
+        apdbSchemaFile = _data_file_name("apdb-schema.yaml", "dax_apdb")
+        with open(apdbSchemaFile) as yaml_stream:
+            table_list = list(yaml.safe_load_all(yaml_stream))
+            for table in table_list:
+                if table['table'] == 'DiaSource':
+                    apdbSchemaColumns = [column['name'] for column in table['columns']]
```

Comment:
Wouldn't you then want a `break` after this, since there should only be one `DiaSource` table, so you no longer need to loop over the table_list?